codelabs/codelab-02/README.md

```diff
@@ -2,7 +2,7 @@
 
 ### Due Date
 
-This codelab is due on *Thursday, February 8th at 11:59:59PM*.
+This codelab comes in two parts, both of which are due on *Thursday, February 15th at 11:59:59PM*. This is part A, it looks long, but a lot of it is just setup + casual walkthrough. Part B will be release 1 week later and cover CloudFront.
```

Comment:
grammar nit:

This is part A which may look long, but most of it is just setup paired with a casual walkthrough. Part B will be released next week and will introduce CloudFront.